pyrdp/player/gdi/__init__.py

```diff
@@ -0,0 +1,7 @@
+#
+# This file is part of the PyRDP project.
+# Copyright (C) 2019 GoSecure Inc.
```

Comment:
2020?

Reply:
Its 2019 everywhere i think

Reply:
You can't bump copyright years everywhere. Proper copyright handling means bumping when you modify the file. So as we change files, we need to update the year. Top-level `LICENSE` file can be updated as soon as one file in the package is.

Reply:
i think its a new file so it should be 2020, right?

Reply:
Yes, it should be 2020. I'll go through the files that are in the PR and make sure it's 2020 everywhere.